http/src/request/channel/update_channel.rs

```diff
@@ -285,7 +285,8 @@ impl<'a> UpdateChannel<'a> {
     fn start(&mut self) -> Result<()> {
         let mut request = Request::builder(Route::UpdateChannel {
             channel_id: self.channel_id.0,
-        });
+        })
+        .json(&self.fields)?;
```

Comment:
What's going on here?

Reply:
What do you mean?

Reply:
oh

Reply:
In this commit I rewrote the function and forgot to keep field serialization https://github.com/twilight-rs/twilight/commit/1d2afb0614d84408c3baf815b0538e017b2ee680#diff-b69469fbd791d47da002c00507d64d16ba1c26e40d975d52ac8e2e68acf88d17L286-R295

Reply:
Glad it's caught